package.json

```diff
@@ -137,8 +137,8 @@
     "lint-staged": "lint-staged",
     "postinstall": "node ./scripts/linkPackages.js",
     "prepublishOnly": "node scripts/prepublishOnly.js",
-    "prettier": "prettier --config .prettierrc.js --write",
-    "prettier:check": "prettier \"packages/**/src/**/*.{js,ts,tsx}\" --check --config .prettierrc.js",
+    "prettier": "prettier \"*.{js,jsx,ts,tsx}\" \"cypress/**/*.{js,jsx,ts,tsx}\" \"packages/**/src/**/*.{js,jsx,ts,tsx}\" --config .prettierrc.js --write",
+    "prettier:check": "prettier \"./*.{js,jsx,ts,tsx}\" \"cypress/**/*.{js,jsx,ts,tsx}\" \"packages/**/src/**/*.{js,jsx,ts,tsx}\" --check --config .prettierrc.js",
```

Comment:
This looked like it could be simplified, so I tried the following:

```
"prettier:check": "prettier **/*.{js,ts,tsx} --check --config .prettierrc.js",
```

The result is:
![image](https://user-images.githubusercontent.com/5121148/93246499-2fa4cb80-f78d-11ea-9500-ece630367ea8.png)

This is **master** branch BTW, so results may be different in v5 branch.

Anyhow, can you try the updated `prettier:check` command maybe and see what results you get?